### DISCUSSION TITLE
VechaiUI starter kit with Remix

### DISCUSSION THREAD

**Discussion by mhaidarhanif** [vechai/vechaiui#70] [Show and tell]
Hello!

I just made a Remix starter kit with Tailwind CSS family of libraries: Headless UI, Radix UI, VechaiUI, and more. It's now called Rewinds.

Check out the repo: https://github.com/mhaidarhanif/rewinds

Preview demo: https://rewinds.vercel.app

Hopefully will help anyone who needs a template with:

- Remix as the React framework
- Tailwind CSS using VechaiUI as primary components
- Theme switcher and persist theme with a session cookie

Thanks!

![Screenshot](https://raw.githubusercontent.com/mhaidarhanif/rewinds/main/public/assets/screenshots/home.png)

**Comment by pepelele** [comment]
Thanks u... It looks really awesome 😸

**Reply by mhaidarhanif** [reply]
Thanks to you too @pepelele 😀
